### PR TITLE
Updates for Minio Server and mc releases

### LINKED ISSUE #955: [RELEASE] mc  RELEASE.2023-08-15T23-03-09Z
mc [RELEASE.2023-08-15T23-03-09Z](https://github.com/minio/mc/releases/tag/RELEASE.2023-08-15T23-03-09Z) has changes that impact the docs:

- [x] Adds `client` command to `mc support perf` ([PR #4645](https://github.com/minio/mc/pull/4645))

### LINKED ISSUE #956: [RELEASE] MinIO Server  RELEASE.2023-08-16T20-17-30Z
MinIO Server [RELEASE.2023-08-16T20-17-30Z](https://github.com/minio/minio/releases/tag/RELEASE.2023-08-16T20-17-30Z) has changes that impact the docs:

- [x] `mc support perf client` added and reveals TX data in output ([PR #17718](https://github.com/minio/minio/pull/17718))
- [x] New drive metrics added ([PR #17854](https://github.com/minio/minio/pull/17854))

### PR DESCRIPTION
**Pull request by djwfyi**
Updates for mc and MinIO server releases
    
- Adds `client` command to mc support perf
- Adds new metrics
- Corrects typos and minor issues in metric related docs
    
Closes #955
Closes #956

### CODE REVIEW

**Comment by djwfyi** [comment]
Staged: 
http://192.241.195.202:9000/staging/grafana/operations/monitoring/metrics-and-alerts.html

File: source/reference/minio-mc/mc-support-perf.rst [hunk 1]
```diff
@@ -148,6 +152,19 @@ Syntax
                       [--airgap]          \
                       ALIAS
 
+.. mc-cmd:: client
+   :fullpath:
+
+   Measure the network throughput from the server to a client.
```
Comment (ravindk89) [inline source/reference/minio-mc/mc-support-perf.rst:158]: Rather than a client, this likely measures perf between the local host at the server, I assume?

**Review by ravindk89 — APPROVED** [review]
One clarifying question, but I do not anticipate the outcome changing an LGTM.

File: source/reference/minio-mc/mc-support-perf.rst (hunk 1 above)
Comment (djwfyi) [reply source/reference/minio-mc/mc-support-perf.rst]: Clarified this with jiuker.
The client is MinIO Client running on the local machine, measuring the rate to talk to the desired server.

Updated the wording accordingly.

**Pull request merged**